eventually/static/src/containers/event/Event.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import {Link} from 'react-router';
+import RaisedButton from 'material-ui/RaisedButton';
+import {getOwner, getTeam} from './EventService';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import EventEdit from '../event/EventEdit';
+import EventTaskList from './EventTaskList';
+
+const STATUS_CHOICES = {
+    0: 'draft',
+    1: 'published',
+    2: 'going',
+    3: 'finished'
+};
+
+const styleMain = {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    width: '100%',
+    maxWidth: '640px',
+    alignItems: 'center',
+    boxShadow: '0px 0px 31px 2px rgba(0,0,0,0.53)',
+    margin: '20px auto',
+};
+
+const styleLowerMain1 = {
+    display: 'flex',
+    justifyContent: 'center',
+    width: '100%',
+    margin: '10px 10px',
+};
+
+const styleInp = {
+    fontSize: '25px',
+    fontFamily: 'Roboto, sans-serif',
+};
+
+const styleSpan = {
+    display: 'inline-block',
+    fontSize: '15px',
+    width: '117px',
+};
+const styleLowerMain2 = {
+    display: 'flex',
+    justifyContent: 'flex-end',
+    width: '85%',
+    margin: '10px 10px',
+};
+
+const buttonStyle = {
```

Comment:
remove buttonStyle as you do not use it anywhere

Reply:
Done